src/OpenTelemetry.Instrumentation.AWSLambda/CHANGELOG.md

```diff
@@ -2,8 +2,13 @@
 
 ## Unreleased
 
-* Change dependency from `OpenTelemetry.Contrib.Extensions.AWSXRay` to `OpenTelemetry.Extensions.AWS`
-  ([#1289](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/1289))
+## 1.2.0-beta.1
+
+Released 2023-Aug-03
+
+* BREAKING: `AddAWSLambdaConfigurations` no longer removes all existing resource attributes
+* BREAKING: Change dependency from `OpenTelemetry.Contrib.Extensions.AWSXRay` to `OpenTelemetry.Extensions.AWS`
+  ([#1289](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/1289)). This now requires at least OpenTelemetry 1.5.1.
```

Comment:
See @utpilla comment https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/1289#discussion_r1282288389

Reply:
Fixed.

Reply:
I think this is relevant enough for users of this library to keep the repetition it here (and the reason for adding BREAKING)